source/extended-json.rst

```diff
@@ -0,0 +1,618 @@
++--------------------+----------------------------------------------------------+
+|MinKey              |{"$minKey": 1}                                            |
++--------------------+----------------------------------------------------------+
+|MaxKey              |{"$maxKey": 1}                                            |
++--------------------+----------------------------------------------------------+
+|Undefined           |{"$undefined": *true*}                                    |
++--------------------+----------------------------------------------------------+
+|Array               |*array*                                                   |
++--------------------+----------------------------------------------------------+
+|Boolean             |*true* or *false*                                         |
++--------------------+----------------------------------------------------------+
+|Null                |*null*                                                    |
++--------------------+----------------------------------------------------------+
+
+.. [#] A *hex string* is a JSON string that contains only hexadecimal
+                digits [0-9a-f] (uppercase is not canonical)
+.. [#] This MUST conform to the `Decimal128 specification`_
+
+.. [#] BSON Regex options MUST be in alphabetical order.
+
+.. [#] See https://docs.mongodb.com/manual/reference/glossary/#term-namespace
+
+.. [#] See https://docs.mongodb.com/manual/reference/database-references/#dbrefs
+
+.. _Decimal128 specification: https://github.com/mongodb/specifications/blob/master/source/bson-decimal128/decimal128.rst#writing-to-extended-json
+
+Representation of Special Numeric Values
+----------------------------------------
+
+Following the `extended JSON format for the Decimal128 type`_, special numeric
+values are encoded as follows:
+
++----------------------------------------+----------------------------------------+
+|**Value**                               |**String**                              |
++----------------------------------------+----------------------------------------+
+|Positive Infinity                       |``Infinity``                            |
++----------------------------------------+----------------------------------------+
+|Negative Infinity                       |``-Infinity``                           |
++----------------------------------------+----------------------------------------+
+|NaN (all variants)                      |``NaN``                                 |
++----------------------------------------+----------------------------------------+
+
+.. _extended JSON format for the Decimal128 type: https://github.com/mongodb/specifications/blob/master/source/bson-decimal128/decimal128.rst#to-string-representation
+
+For example, a BSON floating-point number with a value of positive infinity
+would be encoded as extended JSON as follows::
+
+  {"$numberDouble": "NaN"}
```

Comment:
I think Luke was just referencing existing strings we use for Decimal128 string representations and saying that we will be using those same strings for other numeric types in the extended JSON spec (double, 32-bit integers, and 64-bit integers).

That said, the example is inconsistent with its description. The text says "with a value of positive infinity" and the example is using `"NaN"` when it should probably read `"Infinity"`.